drv/List.drv

```diff
@@ -99,7 +99,31 @@ public interface LIST KEY_GENERIC extends List<KEY_GENERIC_CLASS>, COLLECTION KE
 	 */
 
 	void size(int size);
+	
+	/** Trims this array list so that the capacity is equal to the size.
+	 *
+	 * @see java.util.ArrayList#trimToSize()
+	 */
+	default void trim() {
+		trim(0);
+	}
 
+	/** Trims the backing array if it is too large.
```

Comment:
Backing array makes no sense here. What about e.g. linked lists?

Reply:
Wait they exist in fastUtil?

Reply:
They don't I think, but someone might implement them subclassing this interface. That's the point, its a generic list collection interface, not a array list interface.